pkg/config/config.go

```diff
@@ -35,12 +35,11 @@ import (
 )
 
 const (
-	pprofMemoryTotal       string = "memory_total"
-	pprofBlockTotal        string = "block_total"
-	pprofGoroutineTotal    string = "goroutine_total"
-	pprofMutexTotal        string = "mutex_total"
-	pprofProcessCPU        string = "process_cpu"
-	pprofThreadcreateTotal string = "threadcreate_total"
```

Comment:
removed this as well. as it is not used anymore.

Reply:
why does it show that I removed all the variables 😅

Reply:
whitespace changes...